x-pack/plugins/secops/public/components/timeline/data_providers/icons_footer.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiHorizontalRule, EuiIcon, EuiToolTip } from '@elastic/eui';
+import { isEmpty } from 'lodash/fp';
+import moment from 'moment';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+import { DataProvider } from './data_provider';
+
+const HorizontalBar = styled(EuiHorizontalRule)`
+  margin: 4px 0px;
+`;
+
+const GroupIcons = styled(EuiFlexGroup)`
+  width: 100%;
+`;
+
+interface OwnProps {
+  dataProvider: DataProvider;
+}
+export const IconsFooter = pure<OwnProps>(({ dataProvider }: OwnProps) => {
+  if (!dataProvider.queryDate || (dataProvider.queryDate && isEmpty(dataProvider.queryDate))) {
+    return null;
```

Comment:
If you're using `isEmpty` it will return on `undefined` and `null` so this could just be:

```ts
if (isEmpty(dataProvider.queryDate) {
  return null;
}
```

Reply:
It won't give you the same result